src/main/java/org/broadinstitute/consent/http/service/DataAccessRequestService.java

```diff
@@ -410,6 +413,9 @@ public DataAccessRequest updateByReferenceIdVersion2(User user, DataAccessReques
             dar.getSubmissionDate(),
             now,
             dar.getData());
+        if (Objects.nonNull(dar.getCollectionId())) {
```

Comment:
For the time being, until the mapper bug is fixed:

```suggestion
        if (Objects.nonNull(dar.getCollectionId()) && dar.getCollectionId() > 0) {
```